test/e2e/framework/util.go

```diff
@@ -163,17 +153,13 @@ const (
 	// Minimal number of nodes for the cluster to be considered large.
 	largeClusterThreshold = 100
 
-	// TODO(justinsb): Avoid hardcoding this.
```

Comment:
try separating such changes in separate PRs or separate commits.

Reply:
ok